pkg/commands/run.go

```diff
@@ -327,11 +328,24 @@ func (c *runCommand) runAndPrint(ctx context.Context, args []string) error {
 		}()
 	}
 
+	enabledLintersMap, err := c.dbManager.GetEnabledLintersMap()
+	if err != nil {
+		return err
+	}
+
+	c.printDeprecatedLinterMessages(enabledLintersMap)
+
 	issues, err := c.runAnalysis(ctx, args)
 	if err != nil {
 		return err // XXX: don't lose type
 	}
 
+	// Fills linters information for the JSON printer.
+	for _, lc := range c.dbManager.GetAllSupportedLinterConfigs() {
+		isEnabled := enabledLintersMap[lc.Name()] != nil
```

Comment:
more go way?
```
_, isEnabled := enabledLintersMap[lc.Name()]
```

Reply:
it was existing code.

Reply:
It's not the same thing: the key can exist but the value can be nil.